snow/networking/handler/handler.go

```diff
@@ -874,6 +874,17 @@ func (h *handler) handleChanMsg(msg message.InboundMessage) error {
 		return engine.Notify(context.TODO(), common.Message(msg.Notification))
 
 	case *message.GossipRequest:
+		if state.Type == p2p.EngineType_ENGINE_TYPE_SNOWMAN {
+			avalancheEngine, ok := h.engineManager.Get(p2p.EngineType_ENGINE_TYPE_AVALANCHE).Get(state.State)
+			if ok {
+				// This chain was linearized, so we should gossip the Avalanche
+				// accepted frontier to make sure everyone eventually linearizes
+				// the chain.
+				if err := avalancheEngine.Gossip(context.TODO()); err != nil {
```

Comment:
After some point when the entire network linearizes the DAG will this line still be needed?

Reply:
No, this line is only needed during the linearization.

Reply:
Can we leave a TODO to remove this then?